site/public/examples/examples.json

```diff
@@ -9,6 +9,11 @@
             "name": "SR-Latch Gate Implementation",
             "file": "example2.circuit",
             "thumbnail": "./img/thumbnails/example2.png"
-        }
+        },
+	{
+            "name": "Full Adder Circuit",
+            "file": "Full Adder Circuit.circuit",
+            "thumbnail": "./img/thumbnails/example2.png"
+	}
```

Comment:
Braces look misaligned here, make sure to be using 4-spaces and not tabs because tabs can be evil

Reply:
By here, I mean the braces are misaligned with the rest of the braces in the file

Reply:
Ok will do. Am I using Github correctly? I havent really used it much before.

Reply:
Yeah! All you gotta do now is make the changes and then add/commit the files and push again and it'll automatically update this pull request